### DISCUSSION TITLE
font-custom-4k-smaller 字體大小錯誤回報

### DISCUSSION THREAD

**Discussion by vlkppa** [GpointChen/FFXIVChnTextPatch-GP#296] [General]
任務搜索器和隊員招募視窗字體大小過大，如下圖片示意
<img width="332" alt="ffxiv_20230208_221248_685" src="https://user-images.githubusercontent.com/88968842/217557174-2c9239b4-b154-4193-b065-267ff326889f.png">
<img width="338" alt="ffxiv_20230208_224052_161" src="https://user-images.githubusercontent.com/88968842/217562411-d2b1c756-9f4a-48ba-ac69-94abea0af263.png">

**Comment by GpointChen** [comment]
這個屬於已知的問題，4K字體包在特定解析度和放大倍率組合下會有這樣的現象。

短時間內不會刻意進行調整。

**Comment by xlinshan** [comment]
这些部分的原版字体的宽度会更窄，假如被替换成正常宽度的字体就会这样。我个人的处理方法是在生成字体时，对这几种宽度不同的字体替换设置部分添加一个字体变形，让他们宽度接近，这样替换完后就不会这样。
![image](https://user-images.githubusercontent.com/4050147/224455066-62dd679a-8455-43a0-94fb-a607c9861c4d.png)
类似截图的情况，一些变窄，一些变宽，还有一些更重。
